Add missing attributes that have been added to SRW (#12061)
opensearch-project/documentation-website: _search-plugins/search-relevance/experiments.md

Search Relevance Workbench offers multiple types of experiments. For more information, see [Available search result quality experiments]({{site.url}}{{site.baseurl}}/search-plugins/search-relevance/using-search-relevance-workbench/#available-search-result-quality-experiments).

## Creating experiments

You can create an experiment to test search configurations in various ways.

### Endpoint

```json
POST _plugins/_search_relevance/experiment
```

### Request body fields

The following table lists the available input parameters.

Field | Data type | Description
:--- | :--- | :---
`name` | String | The name of the experiment.
`description` | String | Freeform description of the experiment.

> `type` | String | The type of experiment: `PAIRWISE_COMPARISON`, `POINTWISE_EVALUATION`, or `HYBRID_OPTIMIZER`.

Reviewer: ```suggestion
`type` | String | The experiment type. Valid values are `PAIRWISE_COMPARISON`, `POINTWISE_EVALUATION`, and `HYBRID_OPTIMIZER`.
```
